machine.go

```diff
@@ -867,7 +867,7 @@ func (m *Machine) RunInMachineWithArgs(args []string) (int, error) {
 	executable, err := exec.LookPath(os.Args[0])
 
 	if err != nil {
-		return -1, fmt.Errorf("Failed to find executable: %v\n", err)
+		return -1, fmt.Errorf("Failed to find executable: %w\n", err)
```

Comment:
Seems like the `\n` is redundant & also the error message shouldn't start with a capital (not introduced here of course)

Reply:
I will fix this in a follow-up.